design/ShaderLanguageFAQ.md

```diff
@@ -130,28 +135,24 @@ for handling WGSL, including parsing, validation, and compilation.
 
 At this time, we expect at least three essentially independent
 implementations of WGSL ingestion as part of browser support for WebGPU.
-Independent implementation serves as an important check on the quality
-of the WGSL and WebGPU specifications.
+We expect the existence of these independent implementations to lead to a
+community and culture of competition and innovation, rather than stagnation.
 
 
 ## What about my existing shaders, compilers, and other tooling?
 
 This is also expressed as “the last thing I need is another shader language”.
 (See https://xkcd.com/927/)
 
-For developers invested in the SPIR-V ecosystem for Vulkan and OpenGL,
-the key will be convertibility between WGSL and SPIR-V.
-For shaders that only use WebGPU features, WebGPU Community Group
-members expect to develop tooling to faithfully convert between WGSL
-and “Shader” SPIR-V.
-By “faithful”, we mean that all salient properties of a shader should
```

Comment:
since your commit still uses "faithful", I believe it still needs to define it somehow